index.ts

```diff
@@ -178,7 +180,7 @@ class IGCParser {
   }
 
   get result(): IGCParser.IGCFile {
-    if (!this._result.loggerId) {
+    if (!this._result.loggerId && !this.lenient) {
```

Comment:
I guess we should check for `loggerManufacturer` here, instead of `loggerId`. In that case we wouldn't need to check for `lenient` here for your example file.